File: frontend/app/src/main/java/com/example/nomo/ui/addticket/FriendWithDebt.java
package com.example.nomo.ui.addticket;

import com.example.nomo.model.Friend;

public class FriendWithDebt {
    private Friend friend;
    private String amount = "";
    private boolean isSaved = false;

    public FriendWithDebt(Friend friend) {
        this.friend = friend;
    }

    public Friend getFriend() {
        return friend;
    }

    public String getAmount() {
        return amount;
    }

    public void setAmount(String amount) {
        this.amount = amount;
    }

    public boolean isSaved() {
        return isSaved;
    }

    public void setSaved(boolean saved) {
        isSaved = saved;
    }
}

File: frontend/app/src/main/java/com/example/nomo/ui/addticket/FriendWithDebt.kt
package com.example.nomo.ui.addticket

import com.example.nomo.network.ApiService.Friend

data class FriendWithDebt (
    val friend: Friend,
    var amount: String = "0",
    var isSaved: Boolean = false
)

enum class FriendSelectionMode {
    SINGLE, MULTIPLE
}